src/components/page-layout/page-layout.js

```diff
@@ -38,32 +38,47 @@ export default class PageLayout extends React.PureComponent {
 
   // render the page's content
   renderContent = (config, parentPath, parent, hasSection) => {
-    const { constants, frontMatter, location, domain } = this.props;
+    const { constants, frontMatter, location, domain, navigation } = this.props;
```

Comment:
I'm wondering if we should move breadcrumb data to the navigation batfish helper, but that can definitely be addressed in another PR!

Reply:
Moved this to an issue! https://github.com/mapbox/dr-ui/issues/460